src/docs/user/references/Variables.adoc

```diff
@@ -162,22 +160,37 @@ Examples:
-* *PA:USER_FOLDER* : variable whose value is the relative path of a folder in the <<_global_and_user_spaces, User Data Space>>. At workflow submission, the scheduler server verifies that the folder exists in the user dataspace. Note, the variable value should not end with a slash to avoid the problem of duplicate slash in its usage.
+[none]
+* {empty}
 
+** *PA:CREDENTIAL* : variable whose value is a key of the <<_third_party_credentials, ProActive Scheduler Third-Party Credentials>> (which are stored on the server side in encrypted form). The variable allows the user to access a credential through `credentials.get(variables.get("MY_CRED_KEY"))` instead of in plain-text in the workflow definition. At workflow submission, the scheduler server verifies that the key exists in the 3rd party credentials of the user.
+** *PA:CRON* : variable syntax must be a valid cron expression as defined in the  http://www.sauronsoftware.it/projects/cron4j/manual.php#p02[cron4j manual]. +
+Examples: PA:CRON will accept "5 * * * \*" but not "* * * *" (missing minutes sub-pattern).
+** *PA:DATETIME(format)* , *PA:DATETIME(format)[min,max]* : variable can be converted to a https://docs.oracle.com/javase/7/docs/api/index.html?java/util/Date.html[java.util.Date] using the format specified (see the format definition syntax in the https://docs.oracle.com/javase/7/docs/api/index.html?java/text/SimpleDateFormat.html[SimpleDateFormat] class). +
+A range can also be used in the PA:DATETIME model. In that case, each bound of the range must match the date format used. +
++
+Examples: +
+PA:DATETIME(yyyy-MM-dd) will accept "2014-12-01" but not "2014". +
+PA:DATETIME(yyyy-MM-dd)[2014-01-01, 2015-01-01] will accept "2014-12-01" but not "2015-03-01". +
+PA:DATETIME(yyyy-MM-dd)[2014, 2015] will result in an error during the workflow definition as the range bounds [2014, 2015] are not using the format yyyy-MM-dd.
+** *PA:MODEL_FROM_URL(url)* : variable syntax must match the model fetched from the given URL. This can be used for example when the model needs to represent a list of elements which may evolve over time and is updated inside a file. Such as a list of machines in an infrastructure, a list of users, etc. +
+See <<_variable_model_using_resource_manager_data>> for premade models based on the Resource Manager state. +
+Examples: PA:MODEL_FROM_URL(file:///srv/machines_list_model.txt), if the file machines_list_model.txt contains PA:LIST(host1,host2), will accept only "host1" and "host2", but may accept other values as the machines_list_model file changes.
+** *PA:GLOBAL_FILE* : variable whose value is the relative path of a file in the <<_global_and_user_spaces, Global Data Space>>. At workflow submission, the scheduler server verifies that the file exists in the global dataspace.
```

Comment:
In addition, this model enables the user to graphically browse the global space to select a file at the submission time.